index.jsx

```diff
@@ -15,6 +15,8 @@ export class Radio extends React.Component {
     return (
       <input
         {...this.props}
+        role="radio"
```

Comment:
This role is [implicit ARIA semantics for `input[type=radio]`](https://www.w3.org/TR/html-aria/#input-radio), so should not be used.
And we can use `menuitemradio` as a role for the element. But if you do, we can not overwrite the role.